crates/infra/utils/src/commands/mod.rs

```diff
@@ -131,11 +131,11 @@ fn run_with_defaults(command: &Command) -> Result<()> {
         .wait()
         .with_context(|| format!("Failed to wait for status: {command}"))?;
 
-    return check_status(command, status).map_err(|error| {
+    check_status(command, status).map_err(|error| {
```

Comment:
nit: not sure if this change was intended for this PR?

On a parallel, relating to my earlier suggestion about rust implicit returns, which applies exactly to this instance. I think `return` is meaningful here. It conveys that `check_status()` would process the command and return some value, rather than relying on the trailing `;` in the expression to convey whether this returns a value or not.

Reply:
Removed in fc162fb66b317315b45e3ff9b2de68adbdd431f3, sorry for the omission.

In this particular case, it's somewhat frowned upon to do side-effects (like process exit) in `map` or `map_err`, so this probably should be, removing the trailing-expression-is-return-value effect here.
```rust
if let Err(error) = check_status { ... }
```